addons/project/views/project_views.xml

```diff
@@ -1856,7 +1856,7 @@
                 </xpath>
                 <xpath expr="//div[hasclass('o_kanban_manage_view')]" position="inside">
                     <div role="menuitem" groups="project.group_project_user">
-                        <a name="project_update_all_action" type="object" t-attf-context="{'active_id': #{record.id.raw_value}}">Project Updates</a>
+                        <a name="project_update_all_action" type="object" t-attf-context="{'active_id': #{record.id.raw_value} }">Project Updates</a>
```

Comment:
I guess this was not your intend 🙂

Reply:
Yes it is intended, I had an error from I converted the view in OWL 😅